### ISSUE TITLE
Custom prepublication checklist items are not usable

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#4106]
## Problem
This HTML has the wrong attributes and opens the wrong modal:

https://github.com/BirkbeckCTP/janeway/blob/7882b42a082e096d14719f9a75042d160182bd3c/src/templates/admin/journal/publish_article.html#L156-L172

## Proposed solution
Deprecate this feature (on a quick survey I could not find a large current installation using it).

## Also considered
 Refactor the feature.

**Comment by ajrbyers** [comment]
I don't think we should deprecate this feature, I think we should improve it and make a manager interface to create items so its surfaced more easily.
